pandas/core/indexes/base.py

```diff
@@ -6955,7 +6955,7 @@ def _cmp_method(self, other, op):
                     # TODO: should set MultiIndex._can_hold_na = False?
                     arr[self.isna()] = False
                 return arr
-            elif op in {operator.ne, operator.lt, operator.gt}:
+            elif op == operator.ne:
```

Comment:
is?

Reply:
Was not sure about it, but it does get hit through !=

Reply:
quickly scanning the code, it appears we mostly use `is` elsewhere

Reply:
done